scripts/ci/check/version_test.go

```diff
@@ -28,13 +28,23 @@ import (
 	"golang.org/x/mod/modfile"
 )
 
-const GoVersion = "1.20"
+const (
+	GoVersion = "1.20"
+	CpuType   = "64"
+)
 
 func TestGoVersion(t *testing.T) {
 	goversion, err := exec.Command("go", "version").Output()
```

Comment:
I think we can use `runtime.Version()` here

https://pkg.go.dev/runtime#Version

and use the const GOARCH for arch detection.

Reply:
Oh yes, this seems to be a much better approach than running an `os.exec` command. I'll implement this on a different branch and can merge it later when we come to a consensus.